### DISCUSSION TITLE
Share your Tech videos

### DISCUSSION THREAD

**Discussion by ashwinexe** [EddieHubCommunity/support#2151] [Show and tell]
Making videos are one of the most effective way to improve the **public speaking skills**. 
- You can share Tech videos you made on Youtube, short-video tech tips made on Reels or TikTok, livestreams from any platform.
- The aim here is to spread knowledge while also working on the speaking skills
- Don't hesitate if its your first time (everyone's a beginner at some point)
- Please make sure you are following the community guidelines and code of conduct while sharing.

I'll go first.

**Comment by ashwinexe** [comment]
My first video is focused on web development **beginners** who have no knowledge of it.
- There are several technical and non technical mistakes, stutters, tongue slips which made me realize how much I need to work upon
- This was my **first technical tutorial** (unscripted) and most of it was taken in one go, as I wanted to practice improvising.
- any constructive feedback would be great.

Hope you like it :)

Click below to watch it 👇🏻 
[![Thumbnail](https://user-images.githubusercontent.com/53075480/128353344-da6714b0-9274-4652-a328-3c964da66ea8.png)](https://youtu.be/t8L2eHkKZXk)

**Reply by Sam-Varghese** [reply]
Congratulation! 🎉Nice video. Keep growing 🙂
